docs/source/guide/index.rst

```diff
@@ -217,19 +216,14 @@ Step 2: Load the Online Key
 
     The RSTUF Worker uses this key during the process of managing the metadata.
 
-    Note: the ceremony process won't show any password or key content.
+    Note: It requires the public key information (key id/public hash) only.
 
-    Choose 1/1 ONLINE key type [ed25519/ecdsa/rsa] (ed25519):
-    Enter 1/1 the ONLINE`s private key path: tests/files/online.key
-    Enter 1/1 the ONLINE`s private key password:
-    [Optional] Give a name/tag to the key:
-    ✅ Key 1/1 Verified
+    🔑 Key 1/1 ONLINE
```

Comment:
Not sure why, but step 2 is missing when rendering on GitHub at least:
![image](https://github.com/repository-service-tuf/repository-service-tuf-cli/assets/16246778/964bb40b-4ba7-40c3-9ce3-3c9862b4bc47)

Reply:
Done.